css/80_app.css

```diff
@@ -2958,11 +2958,10 @@ img.tag-reference-wiki-image {
     left: 0;
     right: 0;
     height: 100%;
+    z-index: 100;
     display: block;
     overflow-x: hidden;
     overflow-y: auto;
-    -ms-overflow-style: none;
-    scrollbar-width: none;
```

Comment:
Why did you get rid of these two lines? My intention was to emulate what is done [below](https://github.com/openstreetmap/iD/pull/8791/commits/71728d9415fa132cdd75e6b0afb70dbd1a9d3b3b#diff-3f86fb5cebd04342575e756d440d18a0b6cfb131a3c26d7897ff1793f8b6d30bR2967-R2969) with the `::-webkit-scrollbar` also for [Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width)/IE.

Reply:
Good catch. Naively thought these were redundant. Reverted them back. Thanks.